ChangeLog

```diff
@@ -18,6 +18,8 @@ Bugfix
      Fixes #1040.
    * Fix C89 incompatibility in benchmark.c. Contributed by Brendan Shanks.
      #1353
+   * Return plaintext data sooner on unpadded decryption, as stated in
```

Comment:
@AndrzejKurek Could you please make some small changes to the changelog entry, here and in the backports?

> Return plaintext data sooner on unpadded **CBC** decryption, as stated in **the** mbedtls_cipher_update documentation. Contributed by Andy Leiserson **in** #1180.